keyboards/iris/keymaps/nstickney/keymap.c

```diff
@@ -0,0 +1,174 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Layers
+#define BASE 0 // Base layer
+#define SYMB 1 // Symbols
+#define SYSH 2 // Symbols, shifted
+#define NUMP 4 // Numpad
+#define FCTN 8 // Functions
+
+// Unicode input
+#undef UNICODE_SELECTED_MODES
+#define UNICODE_SELECTED_MODES UC_OSX, UC_LNX, UC_WINC
```

Comment:
The undef and define should actually go in the `config.h` file, or they won't be picked up properly.